include/language-factory.php

```diff
@@ -18,7 +18,26 @@ class PLL_Language_Factory {
 	 *
 	 * @phpstan-var array<string, array<string, string>>
 	 */
-	private static $languages;
+	private $languages;
```

Comment:
Maybe we could keep this property static, thus avoiding to read the languages file as many times as we create instances.

Reply:
Good idea, done in c37ec018.